src/main/java/org/nuxeo/onedrive/client/OneDriveItemIterator.java

```diff
@@ -61,4 +61,8 @@ public boolean hasNext() throws OneDriveRuntimeException {
         return nextMetadata;
     }
 
+    public String getDeltaLink() {
```

Comment:
I would rather having an OneDriveItemDeltaIterator.
We could : 
- add a hook method in JsonObjectIterator, like onResponse(JsonObject json)
- add a protected constructor in OneDriveItemIterator as this OneDriveItemIterator(OneDriveAPI, JsonObjectIterator)
- add OneDriveItemDeltaIterator class extending OneDriveItemIterator and providing an anonymous implementation of JsonObjectIterator which implements onResponse and save the deltaLink.